SuperBuild/External_CTK.cmake

```
@@ -73,7 +73,7 @@ if(NOT DEFINED CTK_DIR AND NOT Slicer_USE_SYSTEM_${proj})
 
   ExternalProject_SetIfNotDefined(
     Slicer_${proj}_GIT_TAG
-    "cd194fff1360e2da114b6b55ce963f089c6f46f2"
+    "aa4a717152052812627a020c7f4110ccb8f7bfc8"
```

To include your latest warnings fixes
```suggestion
    "5d241ecc539dcee182a78b03ecf78514a7fba58c"
```